jenkins_assignment/problem3.txt

```diff
@@ -0,0 +1,17 @@
+pipeline {
+    agent any 
+    stages {
+        stage('Build') { 
+            steps {
+                echo  "stage name: $env.STAGE_NAME" 
+            }
+        }
+        stage('Test') { 
+            steps {
+                  echo  "stage name: $env.STAGE_NAME" 
+
+            }
+        }
+       
+    }
+}
```

Comment:
add the cron job expression used in this exercise (here as a comment or a screenshot)